besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -739,6 +740,7 @@ void setBannedNodeIds(final List<String> values) {
   private BesuConfiguration pluginCommonConfiguration;
   private final Supplier<ObservableMetricsSystem> metricsSystem =
       Suppliers.memoize(() -> PrometheusMetricsSystem.init(metricsConfiguration()));
+  private Vertx vertx;
```

Comment:
nit; `final` modifier?

Reply:
can't :( Get's set outside of constructor (which is required given it depends on metrics, which depends on the cmdline being parsed ... which happens after construction :(